.gitignore

```diff
@@ -49,3 +49,6 @@ node_modules
 
 # Ignore build for client
 /client/build
+
+# Ignore all yarn errors
+yarn-error.log
```

Comment:
I think adding in this form would ignore all files with this name, including those in the `client` folder?